### ISSUE TITLE
Pro controller

### ISSUE THREAD

**Issue by Kilatif** [dekuNukem/Nintendo_Switch_Reverse_Engineering#67]
Hi! I hope you still read this questions section :) I would like to ask, do you explore Pro Controller or Joycons only? Do you have some discord channel maybe where we can talk?

**Comment by v1993** [comment]
Yeah, I'm interested in Pro controller a lot too. Would be glad to see some research!

**Comment by Kilatif** [comment]
@v1993 I can say after long research that in this repo you can find anything for any type of controller (Pro and JoyCons) and you know, Pro and JoyCons very very similar in they protocols.
